pkgs/tools/misc/nginx-config-formatter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2016-06-16";
+  name = "nginx-config-formatter-${version}";
```

Comment:
If I understand correctly it will check the version if you installed the package using the package name (`nix-env -i xyz`) and not the attribute name (`nix-env -iA xyz`).

I'm not sure, though. I don't use the `nix-env -u` mechanism at all so I've never been bitten by this.

In any case I think it is best to change the package name now when very few will be affected. It'll be more painful over time.